utils/targetint.mli

```diff
@@ -0,0 +1,181 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                        Nicolas Ojeda Bar, LexiFi                       *)
```

Comment:
Since this interface file is 95% identical to stdlib/nativeint.mli, please keep the original author of stdlib/nativeint.mli (myself) and add yourself after.